blogs/2022/09/15/dev-container-features.md

```diff
@@ -0,0 +1,139 @@
+
+> **Note:** You should be sure that feature packs you release publicly check and install dependencies in addition to the feature. Furthermore, public features are very likely to be used from both arm64 or x86_64 machines - so be sure to adapt to this when possible.
+
+You may review `devcontainer-feature.json` properties [in the spec](https://containers.dev/implementors/features/#devcontainer-feature-json-properties), along with public examples in the [devcontainers/features](https://github.com/devcontainers/features) repo.
+
+Now that we’ve seen how to create a feature pack, how can I distribute it to others?
+
+## Distribution
+
+Feature packs are distributed as tarballs. The tarball contains the entire contents of the feature pack's sub-directory, including the `devcontainer-feature.json`, `install.sh`, and any other files in the directory.
+
+The [Open Container Initiative](https://opencontainers.org/), aka OCI, defines industry standards for containers and container resources. We treat feature packs as OCI Artifacts and use the concept of an [OCI Registry](https://containers.dev/implementors/features-distribution/#oci-registry) to distribute them.
+
+The [features template repository](https://github.com/devcontainers/feature-template) mentioned above includes a [GitHub Actions workflow](https://github.com/marketplace/actions/dev-container-publish) to automate the publishing process. It packages each feature into a tarball and publishes the assets as an OCI artifact to GHCR. Trigger the `release.yaml` workflow from the template repository by selecting it on the left of the repository’s Actions tab on GitHub. This will publish each feature to GHCR under the `<owner>/<repo>` namespace. A feature pack is only republished when the version property in its `devcontainer-feature.json` is updated.
+
+> **Note:** One manual step with GHCR is to mark the OCI [package as "public"](https://github.com/devcontainers/feature-template#marking-feature-public). This only has to be done once per feature pack. Private feature packs do not require this step and may be accessed as long as you’ve logged into the Docker CLI using the credentials for your registry.
+
+### Sharing your features with the community
+
+If you'd like your contributions to appear in the [VS Code Remote-Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) or [GitHub Codespaces](https://github.com/features/codespaces) UI for dev container creation, you may do the following:
+
+* Go to [devcontainers.github.io](https://github.com/devcontainers/devcontainers.github.io) (the GitHub repo backing [containers.dev](https://containers.dev))
+* Open a PR to modify the [`collection-index.yml`](https://github.com/devcontainers/devcontainers.github.io/blob/gh-pages/_data/collection-index.yml) file
+
+![Feature collections page of containers.dev](./features-collections.png)
```

Comment:
This image is low res and only shows one collection. Maybe it can be removed and just include a link to the https://containers.dev/collections?

Reply:
Updated!

Reply:
Still need to remove the image Markdown

Reply:
Thanks for catching, updated!